app/views/volunteers/_navbar.html.erb

```diff
@@ -0,0 +1,11 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+  <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
```

Comment:
None of these seem to be used. I was able to delete this line (and matching closing tag) without changing the formatting at all.